CMakeLists.txt

```diff
@@ -850,6 +850,11 @@ endif()
 if (Tasmanian_ENABLE_FORTRAN)
     add_test(Fortran fortester)
 endif()
+set(Tasmanian_TESTS_OMP_NUM_THREADS 4 CACHE STRING "Number of OpenMP threads to use in tests (default is 4)")
```

Comment:
Let's talk about the purpose of this tomorrow. We should meet and discuss many other things going forward.